src/ide/ide.ts

```diff
@@ -2524,7 +2524,7 @@ export let ide = (function () {
 			e = createTypedEvent("canvas.KeyboardEvent", e);
 			module.interpreter.enqueueEvent("canvas.keyup", e);
 		});
-		/*
+
```

Comment:
Was it intentional to uncomment this?

Reply:
Not sure tbh, I intended to only uncomment the code that's necessary for the tutorial.

Reply:
Well, I'm asking because the title only says something about unit tests and this doesn't seem to be related...?

Reply:
Yes, I thought the tutorial had already been added but there hasn't been a PR to do that. Changed the title accordingly.